src/OrbitClientData/ModuleManagerTest.cpp

```diff
@@ -131,6 +131,16 @@ TEST(ModuleManager, AddOrUpdateModules) {
   ModuleSymbols module_symbols;
   module->AddSymbols(module_symbols);
   ASSERT_TRUE(module->is_loaded());
+  module_info.set_name("changed name 2");
+
+  unloaded_modules = module_manager.AddOrUpdateModules({module_info});
+  ASSERT_EQ(unloaded_modules.size(), 1);
+  EXPECT_EQ(unloaded_modules[0]->name(), "changed name 2");
```

Comment:
In addition
EXPECT_EQ(unloaded_modules[0], module);

Reply:
done.